### DISCUSSION TITLE
Client Unix socket ????

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#7152] [Q&A]
Here is a little chuck of LiquidSoap log showing entries regarding "New Client Unix Socket" requests which occurs often and always gets disconnected.
Anyone knows what's up with that?

[2024/05/27 17:54:12 [harbor:4] Request to update metadata for mount / on port 8005
2024/05/27 17:54:12 [lang:3] API auth - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/auth' with body: { "user": "Speedy", "password": "DELETED" }
2024/05/27 17:54:12 [lang:3] API auth - Response (200): true
2024/05/27 17:54:12 [harbor:4] Client logged in.
2024/05/27 17:54:12 [input_streamer:3] New metadata chunk ? -- Pierre Guillemette - Vous et moi on est semblable - Speedy en ondes.
2024/05/27 17:54:16 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: {
2024/05/27 17:54:16 [lang:3]   "artist": "",
2024/05/27 17:54:16 [lang:3]   "title": "Pierre Guillemette - Vous et moi on est semblable - Speedy en ondes"
2024/05/27 17:54:16 [lang:3] }
2024/05/27 17:54:16 [lang:3] API feedback - Response (200): true
2024/05/27 17:55:01 [server:4] New client unix socket "".
2024/05/27 17:55:01 [server:4] Client unix socket "" disconnected.
2024/05/27 17:56:02 [server:4] New client unix socket "".
2024/05/27 17:56:02 [server:4] Client unix socket "" disconnected.
2024/05/27 17:57:01 [server:4] New client unix socket "".
2024/05/27 17:57:01 [server:4] Client unix socket "" disconnected.
2024/05/27 17:57:35 [harbor:4] New client on port 8005: DE.LE.TED
2024/05/27 17:57:35 [harbor:4] Method: GET, uri: /admin.cgi?pass=Speedy:Chuckw00d&mode=updinfo&song=Paul%20Brunelle%20-%20Un%20par%20un%20-%20Speedy%20en%20ondes&url=%26artist%3DPaul%2520Brunelle%26title%3DUn%2520par%2520un%26album%3DStars%2520du%2520Country%2520Vol.%25202%26duration%3D164415%26songtype%3DS%26overlay%3DNO%26buycd%3D%26website%3D%26picture%3D, protocol: HTTP/1.0
2024/05/27 17:57:35 [harbor:4] Header: User-Agent, value: Mozilla/4.0.
2024/05/27 17:57:35 [harbor:4] Header: Host, value: stream.radiodesfestivals.ca:8005.
2024/05/27 17:57:35 [harbor:4] http/unix GET request on /admin.cgi.
2024/05/27 17:57:35 [harbor:4] http/unix Arg: song, value: Paul Brunelle - Un par un - Speedy en ondes.]

Thanks for any insights;
Speedy

**Comment by BornKillaz** [comment]
Hi. I'm having the same issue. Updating Azuracast is always a big adventure...

**Reply by SpeedyCharly** [reply]
But I'm on stable release, such issues shouldn't popup at random...

**Comment by BusterNeece** [comment]
What makes you think this is an error? I suspect this is just how Liquidsoap logs the Telnet commands when we connect to issue commands (i.e. to skip a track or disconnect a streamer, etc.).

No reason to think it's causing any trouble, so I don't get why you'd immediately jump to the complaining so quickly.

**Reply by SpeedyCharly** [reply]
Not really complaining - wanted to know what it is...
